### PR TITLE
incorporate Azalee's suggested text (episode 5)

### PR DESCRIPTION
**Pull request by ErinBecker**

### CODE REVIEW

File: _episodes/05-join.md [hunk 1]
```diff
@@ -368,7 +367,11 @@ src="https://github.com/datacarpentry/astronomy-python/raw/gh-pages/fig/join.png
 ## Joining tables
 
 Now let's get to the details of performing a `JOIN` operation.
-As a starting place, let's go all the way back to the cone search from Lesson 2.
+
+We are about to build a complex query using software that doesn’t provide us with much information on debugging.
+For this reason we are going to start with a simplified version of what we want to do until we’re sure we’re joining
+the tables correctly, then we’ll slowly add more layers of complexity, checking at each stage that our query still works.
```
Comment (rudyphd) [inline _episodes/05-join.md:373]:
```suggestion
the tables correctly, then we’ll slowly and add more layers of complexity, checking at each stage that our query still works.
```

File: _episodes/05-join.md [hunk 2]
```diff
@@ -528,9 +533,12 @@ WHERE 1=CONTAINS(
 
 Now that there's more than one table involved, we can't use simple
 column names any more; we have to use **qualified column names**.
-In other words, we have to specify which table each column is in.
+In other words, we have to specify which table each column is in. 
+The column names do not have to be the same and in fact in the next join they will not be.
```
Comment (rudyphd) [inline _episodes/05-join.md:537]:
```suggestion
The column names do not have to be the same and, in fact, in the next join they will not be. 
```

File: _episodes/05-join.md [hunk 3]
```diff
@@ -528,9 +533,12 @@ WHERE 1=CONTAINS(
...
 Now that there's more than one table involved, we can't use simple
 column names any more; we have to use **qualified column names**.
-In other words, we have to specify which table each column is in.
+In other words, we have to specify which table each column is in. 
+The column names do not have to be the same and in fact in the next join they will not be. 
+That is one of the reasons that we explicitly specify them.
 Here's the complete query, including the columns we want from the Gaia
-and best neighbor tables.
+and best neighbor tables. Here you can start to see that using the abbreviated names
+is making our query easier to read and is less typing for us.
```
Comment (rudyphd) [inline _episodes/05-join.md:541]:
```suggestion
is making our query easier to read and requires less typing for us.
```

File: _episodes/05-join.md [hunk 4]
```diff
@@ -893,6 +903,7 @@ interested in the final match, using both criteria.
 
 ## Transforming coordinates
 
+As always we retrieve coordinates from the database in ICRS coordinates but we always want to visualize them in GD-1 coordinates.
```
Comment (rudyphd) [inline _episodes/05-join.md:906]:
```suggestion
As always we retrieve coordinates from the database in ICRS coordinates but we always want to visualize them in GD1 coordinates. 
```

I notice we are mixing GD-1 and GD1, is there a reason or should be make it consistent?

File: _episodes/05-join.md [hunk 5]
```diff
@@ -112,13 +112,12 @@ hope) in the Pan-STARRS catalog.
 
 To do that we will:
 
-1. Use the `JOIN` operator to look up each `source_id` in the
-`panstarrs1_best_neighbour` table, which contains the `obj_id` of the
-best match for each star in the Gaia catalog; then
+1. Use the `JOIN` operator to look up each Pan-STARRS `obj_id` for the stars 
+we are interested in in the` panstarrs1_best_neighbour` table using the `source_id`s
```
[the remark this answers is not in the document]
Comment (ErinBecker) [reply _episodes/05-join.md:116]: I think the double "in" is correct here. We're looking for the stars we are interested in in the table.

**Pull request merged**